src/Components/CardContainer.js

```diff
@@ -25,10 +24,11 @@ class CardContainer extends Component {
 }
 
     render() {
+      console.log(this.props.filteredParades)
       return(
           <div>
-           <section>
-            {this.displayLocationCards()}
+           <section className = "card-container">
+            {this.displayCards()}
```

Comment:
Nice job breaking this rendered portion of the JSX out into a method -- this is a common approach for dealing with large chunks of conditional rendering. Not entirely sure it's 100% necessary here, but you will see this pattern in many places in React apps.